airbyte-integrations/bases/base-normalization/integration_tests/test_ephemeral.py

```diff
@@ -61,8 +61,18 @@ def setup_test_path(request):
 
 
 @pytest.mark.parametrize("column_count", [1500])
-@pytest.mark.parametrize("integration_type", list(DestinationType))
-def test_destination_supported_limits(integration_type: DestinationType, column_count: int, setup_test_path):
+@pytest.mark.parametrize(
+    "integration_type",
+    [
+        DestinationType.POSTGRES,
+        DestinationType.BIGQUERY,
+        DestinationType.SNOWFLAKE,
+        DestinationType.REDSHIFT,
+        # MySQL: the max number of columns is limited by row size (8KB),
+        # not absolute column count. It is way fewer than 1500.
+    ],
```

Comment:
This is not going over the full list of values of DestinationType enum anymore, so if we add a new destination, how do we make sure we don't forget to test it with maximum numbers of columns?

Would it be better to keep the parameter as `list(DestinationType)` but add an `if destination = MySQL then return` in the function code to skip the test for MySQL only?

Reply:
Good point. Done.